src/apps/Casa/README.md

```diff
@@ -42,7 +42,7 @@ const CasaExample = () => {
   });
 
   zoneLayer.onClick(f => {
-    alert(`Clicked on ${f.length} ZoneLayer features.`);
+      console.log('Clicked', f);
```

Comment:
Have you linted this Readme?

Reply:
This is for demo reasons and has to be there.

Reply:
console.log() is okay for demo. I meant the spaces and all the other linting errors. I will accept them for the demo, too.